test/utils.js

```diff
@@ -48,18 +46,18 @@ export function expectSuccessfulTransfer({
     balances[stepFromIndex] -= parseFloat(transaction.value);
     balances[stepToIndex] += parseFloat(transaction.value);
 
-    const { capacity } = edges.find((edge) => {
-      return (
+    edges.forEach((edge) => {
+      if (
         edge.from === transaction.from &&
         edge.to === transaction.to &&
         edge.token === transaction.token
-      );
+      ) {
+        if (Number(edge.capacity) < Number(transaction.value)) {
```

Comment:
although it's true that this is only used for tests, and I'm not sure if the dependency will be installed also when we import the package in other project. If that's the case, it's better to not install web3 to make it lighter. Also, because the tests are dealing with small numbers. 
Sorry for the messing around @JacqueGM  😛